content/docs/ca/contributing/documentation.md

```diff
@@ -1,2 +1,33 @@
 # Contribute to this documentation
 This documentation is a work-in-progress and *you are welcome to pitch in and help*. Simply fork the [Drupal Console Book](https://github.com/hechoendrupal/drupal-console-book "(i.e. this documentation)") on GitHub. If you haven't yet contributed to a project on GitHub, or aren't still sure what the workflow looks like, read the documentation about [pull requests](https://help.github.com/articles/using-pull-requests/). You may also wish to download the GitHub application ([Mac](https://mac.github.com) | [Windows](https://windows.github.com), which simplifies the workflow a bit and provides a nice GUI for your contributions).
```

Comment:
@harold20 link is missing here

Reply:
- Link
https://github.com/hechoendrupal/drupal-console-book

- Should be 
https://github.com/hechoendrupal/drupalconsole.com